R/helpers.R

```diff
@@ -420,13 +420,11 @@ html <- function(text, ...) {
   htmltools::HTML(text, ...)
 }
 
-is.html <- function(x) {
-
-  if (inherits(x, "html")) {
-    isTRUE(attr(x, "html"))
-  } else {
-    FALSE
-  }
+#' Does an object have the `html` class?
+#'
+#' @noRd
+is_html <- function(x) {
+  ifelse(inherits(x, "html"), TRUE, FALSE)
```

Comment:
Is there a reason this isn't just `inherits(x, "html")`?

Also, I'd avoid `ifelse` unless you specifically are doing a _vectorized_ conditional; if it's just about saving some curly braces, I'd still use `if`/`else` since the semantics are more sensible.

Reply:
This is now simplified (😬) and your advice on `ifelse()` is good (and I've remember being burned with improper `ifelse()`s before).